packages/hash/shared/src/types.ts

```diff
@@ -225,7 +225,7 @@ export type SystemPropertyTypeTitle =
 /**
  * The system link type titles.
  */
-const systemLinkTypes = {
+const systemEntityLinkTypes = {
```

Comment:
As mentioned before, perhaps `LinkEntityTypes` and adjusting the comment above this as well